mvp-0/README.md

```diff
@@ -1,117 +1,239 @@
 # MVP-0
 
-## Deployment
-
-### Observer cluster
-
-#### Deploy dnation-kubernetes-monitoring-stack without dnation-kubernetes-monitoring
-
-1. Optional: Fill `thanos-storage.yaml` with bucket credentials in **thanosStorage config**
-2. Deploy
-   ```bash
-   helm repo add dnationcloud https://dnationcloud.github.io/helm-hub/
-   helm repo update dnationcloud
-   helm upgrade --install dnation-kubernetes-monitoring-stack dnationcloud/dnation-kubernetes-monitoring-stack \
-     --set dnation-kubernetes-monitoring.enabled=false \
-     -f values-observer.yaml \
-     -f thanos-storage.yaml # Optional
-   ```
-
-#### Deploy dnation-kubernetes-monitoring
-
-1. Clone and build rules/dashboards
-   ```bash
-   git clone https://github.com/dNationCloud/kubernetes-monitoring.git
-   cd kubernetes-monitoring
-   git checkout scs-kaas-mvp
-   make jsonnet-package
-   cd -
-   ```
-2. Deploy
-   ```bash
-   helm upgrade --install dnation-kubernetes-monitoring kubernetes-monitoring/chart --dependency-update \
-     --set releaseOverride=dnation-kubernetes-monitoring-stack \
-     -f values-observer-dash.yaml
-   ```
-
-#### Deploy kaas-metric-importer
-
-Deployment uses an image built from https://github.com/m3dbx/prometheus_remote_client_golang.
+This MVP-O aims to build an Observer monitoring solution intended to offer a global **metrics**
+view of the CSP infrastructure. It is the platform where CSP infrastructure **metrics**
+are fetched, processed, stored, and visualized. Note that this monitoring solution could
+be extended, and the other two observability signals (logs and traces) from the CSP
+infrastructure could also be processed here.
+
+_This MVP-0 builds a base for an Observer monitoring solution intended to become an **SCS product**
+in the future versions, once it attains sufficient stability._
```

Comment:
I think we should also add a sentence with a warning regarding the usage of this stuff in production systems, what do you think?

Reply:
I agree, 79b0c3d76115c2278b04fe880530c25cb6a4d7fd